validator/src/commands/run/execute.rs

```diff
@@ -203,35 +193,7 @@ pub fn execute(
         None
     };
 
-    let column_options = LedgerColumnOptions {
-        compression_type: match matches.value_of("rocksdb_ledger_compression") {
-            None => BlockstoreCompressionType::default(),
-            Some(ledger_compression_string) => match ledger_compression_string {
-                "none" => BlockstoreCompressionType::None,
-                "snappy" => BlockstoreCompressionType::Snappy,
-                "lz4" => BlockstoreCompressionType::Lz4,
-                "zlib" => BlockstoreCompressionType::Zlib,
-                _ => panic!("Unsupported ledger_compression: {ledger_compression_string}"),
-            },
-        },
-        rocks_perf_sample_interval: value_t_or_exit!(
-            matches,
-            "rocksdb_perf_sample_interval",
-            usize
-        ),
-    };
-
-    let blockstore_options = BlockstoreOptions {
-        recovery_mode,
-        column_options,
-        // The validator needs to open many files, check that the process has
-        // permission to do so in order to fail quickly and give a direct error
-        enforce_ulimit_nofile: true,
-        // The validator needs primary (read/write)
-        access_type: AccessType::Primary,
-        num_rocksdb_compaction_threads: rocksdb_compaction_threads,
-        num_rocksdb_flush_threads: rocksdb_flush_threads,
-    };
+    let blockstore_options = run_args.blockstore_options;
```

Comment:
No need for this IMO, just use it directly below:
```
blockstore_options: run_args.blockstore_options,
```

Reply:
sure! [9ec842d](https://github.com/anza-xyz/agave/pull/6918/commits/9ec842d654ce7ae4d3f2c8b54d4fe55c6e6aae4a)